platform/osgi-platform/src/main/resources/META-INF/services/javax.validation.spi.ValidationProvider

```diff
@@ -0,0 +1 @@
+org.apache.bval.jsr303.ApacheValidationProvider
```

Comment:
Since this is now automatically imported from every bundle, can we drop this file from MDS or should we keep it there for whatever reason?